src/dc_tools.rs

```diff
@@ -586,6 +586,16 @@ pub(crate) fn listflags_has(listflags: u32, bitindex: usize) -> bool {
     (listflags & bitindex) == bitindex
 }
 
+/// Makes sure that a user input that is not supposed to contain newlines does not contain newlines.
+pub(crate) fn improve_single_line_input(input: impl AsRef<str>) -> String {
+    input
```

Comment:
i suggest to use `split_whitespace().collect()` instead - that would also match for tabs and other stuff and it will not just trim the resulting string but also convert `foo\n\rbar` to `foo bar` (instead of leaving two spaces between "foo" and "bar")

Reply:
Actually this will remove all spaces (`foo bar` -> `foobar`).

Reply:
you're right, the collect() would need to be replaced by sth. else. but no worries, it is okay as it is. thanks for merging!